packages/core/src/di/r3_injector.ts

```diff
@@ -73,6 +73,7 @@ import {
 import {ProviderToken} from './provider_token';
 import {INJECTOR_SCOPE, InjectorScope} from './scope';
 import {setActiveConsumer} from '@angular/core/primitives/signals';
+import * as di from '@angular/core/primitives/di';
```

Comment:
Since you're only using `Injector`, why not just import just that class?

```suggestion
import {Injector as PrimitivesInjector} from '@angular/core/primitives/di';
```

Reply:
done